tests/k8st/test_base.py

```diff
@@ -35,15 +35,13 @@ class TestBase(TestCase):
     """
     Base class for test-wide methods.
     """
-    @classmethod
```

Comment:
Unless I'm missing something, I think this was broken, since it was trying to use fields that weren't instantiated until `setUp` - namely, the k8s client.